core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -5929,6 +5929,7 @@ class ReplicaManagerTest {
         Uuid.randomUuid.toString,
         future,
         topicPartitions,
+        0,
```

Comment:
This is `ShareAcquireMode.BATCH_OPTIMIZED.id()` or similar I think. We should have a consistent way of referring to this constant in the broker code and tests.